webapp/.storybook/preview.js

```diff
@@ -0,0 +1,3 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+}
```

Comment:
As mentioned in the lecture, this can be removed.
```suggestion
export const parameters = {

}
```